src/components/TabLinks.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from "react";
+import { Tabs } from "@canonical/react-components";
+import { useNotify } from "@canonical/react-components";
+import { useNavigate } from "react-router-dom";
+import { slugify } from "util/slugify";
+
+interface Props {
+  TABS: string[];
+  activeTab?: string;
+  tabUrl: string;
+}
+
+const TabLinks: FC<Props> = ({ TABS, activeTab, tabUrl }) => {
```

Comment:
The prop naming can be improved, how about `tabs, activeTab, baseUrl`? Prop names should always be `camlCase`.